src/PrestaShopBundle/Resources/config/services/bundle/form/form_extension.yml

```diff
@@ -16,6 +16,22 @@ services:
     tags:
       - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\MoneyType }
 
+  PrestaShopBundle\Form\Admin\Type\CustomNumberType:
+    public: false
+    class: 'PrestaShopBundle\Form\Admin\Type\CustomNumberType'
+    arguments:
+      - '@prestashop.core.localization.locale.context_locale'
+    tags:
+      - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\NumberType }
+
+  PrestaShopBundle\Form\Admin\Type\CustomIntegerType:
+    public: false
+    class: 'PrestaShopBundle\Form\Admin\Type\CustomIntegerType'
+    arguments:
+      - '@prestashop.core.localization.locale.context_locale'
+    tags:
+      - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\IntegerType }
+
```

Comment:
I think you can delete 'class' properties because FQCN is the same, no ? Do you think autowire can work (then we can delete 'arguments' properties too ?